basic-steps/src/main/resources/org/jenkinsci/plugins/workflow/steps/CoreStep/help.html

```diff
@@ -0,0 +1,9 @@
+<div>
+    <p>
+    This is a special step that allows to call build wrappers (also called "Environment Configuration" in freestyle or similar projects).
```

Comment:
No, that is `CoreWrapperStep`. This one is for “build steps” and “post-build actions” (IIRC).